vite.config.js

```diff
@@ -6,5 +7,20 @@ import react from '@vitejs/plugin-react';
 // https://vitejs.dev/config/
 export default defineConfig({
   root: resolve(import.meta.dirname),
-  plugins: [viteFastify({ spa: true }), react()],
+
+  plugins: [
+    viteFastify({ spa: true }),
+    react(),
+    sentryVitePlugin({
+      org: process.env.SENTRY_ORG,
+      project: process.env.SENTRY_PROJECT || 'ui-frontend',
```

Comment:
```suggestion
      project: process.env.SENTRY_PROJECT,
```
Let’s rely on our env variables – alternatively provide defaults for all the other env variables as well but I’d keep it empty.